src/nvim/ex_cmds2.c

```diff
@@ -892,7 +899,14 @@ void ex_drop(exarg_T *eap)
       // execute [+cmd]
       if (eap->do_ecmd_cmd) {
         bool did_set_swapcommand = set_swapcommand(eap->do_ecmd_cmd, 0);
-        do_cmdline(eap->do_ecmd_cmd, NULL, NULL, DOCMD_VERBOSE);
+        exarg_T ea = {
+          .cmd = eap->do_ecmd_cmd,
```

Comment:
Seems like this could have a "lifetime" issue. `eap->do_ecmd_cmd` may get freed and the new `ea.cmd` might not expect that?

`XFREE_CLEAR` may be useful.